Majasdarbs/03-majasdarbs/CV/ilze-mitre.html

```diff
@@ -0,0 +1,213 @@
+    <section id="licenses-certifications">
+      <h2>Licenses & certifications</h2>
+      <ul>
+        <li>Learn CSS Course - Codecademy, Dec 2021</li>
+        <li>Build Websites from Scratch with HTML & CSS - Udemy, Sep 2021</li>
+        <li>Recruiting and Hiring (Version 3) - Eazl, Aug 2021</li>
+      </ul>
+    </section>
+    <!-- Licenses & certifications end-->
+
+    
+    <!-- Skills-->
+    <section id="skills">
+      <h2>Skills & Languages</h2>
+      <b>My most rated skills</b>
+      <ul>
+        <li>Project Management</li>
+        <li>Project Planning</li>
+        <li>Project Coordination</li>
+        <li>Project Proposal Writing</li>
+        <li> Office Administration</li>
+        <li>Human Resources</li>
+        <li>Recruiting</li>
+        <li>Interviewing</li>
+        <li>Payroll Adminsitration</li>
+      </ul>
+      <hr>
+
+      <b>Skills I'm improving now</b>
+      <ul>
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JavaScript</li>
+      </ul>
+      <hr>
+
+      <b>Language knowledge</b>
+      <ul>
+        <li>Latvian</li>
+        <li>English</li>
+        <li>Russian</li>
+      </ul>
+
+      </ul>
+    </section>
+    <!-- Skills end-->
+
+    
+    <!--FTA-->
+    <section id="fta">
+      <h2>Free time activities</h2>
+      <details>
+        <summary>Hobbies</summary>
+        <ul>
+          <li>Travelling with my family</li>
+          <li>Training my dog</li>
+        </ul>
+      </details>
+      <details>
+        <summary>Music for focus</summary>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/sXC6AUbY69A" title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen>
+        </iframe>
+      </details>
+    </section>
+    <!--FTA end-->
+
+    
+    <!--Projects-->
+    <section id="projects">
+      <h2>Projects</h2>
+      <p>To be continued...</p>
+    </section>
+    <!--Projects end-->
+
+    
+    <!-- Contact me-->
+    <section id="contact-me">
+      <h2>Contact me </h2>
+
+      <form action="action_page.php">
+        <input type="text" id="firstname" name="firstname" placeholder="Your name.." style="width: 200px;">
+        <br><br>
+
+        <input type="text" id="company" name="lastname" placeholder="Company name.." style="width: 200px;">
+        <br><br>
+
+        <textarea id="subject" name="subject" placeholder="Write something.."
+          style="width: 200px; height:100px"></textarea>
+        <br><br>
+
+        <input type="submit" value="Submit">
+      </form>
+    </section>
+    <!-- Contact me end-->
+
+  </main>
+
+  
+
+  <footer>
+    <nav>
+      <ul id="my-links">
+        <li><a href="https://github.com/IlzeiMilzei" target="blank">Github</a></li>
+        <li><a href="https://www.linkedin.com/in/ilze-mitre-b4631956/" target="blank">Linkedin</a></li>
```

Comment:
probably should be _blank https://www.w3schools.com/tags/att_a_target.asp

Check the difference between blank and _blank https://css-tricks.com/targetblank/#:~:text=The%20difference%3A,that%20same%20newly%2Dopened%20tab.